src/systems/bit-interaction-system.js

```diff
@@ -5,11 +5,9 @@ import { dontHoldWithHandAndRemote, dontHoverAndHold } from "./not-hovered-if-he
 export function interactionSystem(world, cursorTargettingSystem, t, systems) {
   cursorTargettingSystem.tick(t); // handles hovers for cursors
   handHoverSystem(world, systems.interaction);
-
   holdSystem(world, systems.userinput);
   dontHoldWithHandAndRemote(world);
-  dontHoverAndHold(world);
-
+  // dontHoverAndHold(world); // TODO: Is this necessary?
```

Comment:
We now turned this back on since the original reason we removed it is gone.

I did some basic testing in VR things all seem to work fine. Entities loaded with the new loader as expected can only be grabbed with one hand, not two for scaling. Also videos can't be play/paused since we only implemented it so far for desktop. This is expected.